core/src/agent/keys.rs

```diff
@@ -1,11 +1,9 @@
-use common::hash::Hash;
-
 #[derive(Clone, Debug, PartialEq)]
 pub struct Key {}
 
 #[derive(Clone, Debug, PartialEq)]
 pub struct Keys {
     pub_key: Key,
     priv_key: Key,
-    node_id: Hash,
+    node_id: String,
```

Comment:
We cant use a 128 bit integer because its sha256.  But we could increase efficiency by doing two int compares instead of a string compare.  However, I think this lives in the realm of optimization, and maybe we make a ticket about that?  It also relates to the idea of generalizing out the hashing algorithm see #104